examples/light-switch-app/silabs/SiWx917/args.gni

```diff
@@ -18,6 +18,7 @@ import("${chip_root}/src/platform/silabs/SiWx917/args.gni")
 
```

Comment:
Shouldn't the same modifications are required for the other apps like lighting-app